tests/scripts/remote_monitoring_tests/helpers/utils.py

```diff
@@ -138,43 +138,20 @@
     "container_image_name": "kruize/tfb-qrh:1.13.2.F_et17",
     "container_name": "tfb-server",
     "cpuRequest_name": "cpuRequest",
-    "cpuRequest_sum": 4.4,
-    "cpuRequest_avg": 1.1,
     "cpuRequest_format": "cores",
     "cpuLimit_name": "cpuLimit",
```

Comment:
You should not remove cpu and memory usage values as these are used in other tests